### ISSUE TITLE
Adding reference to sharp in GraphQL query breaks develop + build

### ISSUE THREAD

**Issue by benhodgson87** [birkir/gatsby-source-prismic-graphql#76]
I'm fairly new to Gatsby, so this _could_ just be a lack of understanding, but have been banging my head against the wall trying to resolve this.

If I add my GraphQL query without requesting the imageSharp values, and just pass the url of the image into `<img src={image.url} ...` everything works fine.

As soon as I add imageSharp to the GraphQL query, the development environment just returns a blank page. No terminal errors, no console errors.

Example query with sharp added:
```js
... on PRISMIC_LandingpageBodySplit_content {
  type
  primary {
    title
    footer_cta_text
  }
  fields {
    copy
    image
    imageSharp {
      childImageSharp {
        fluid(maxWidth: 400) {
          ...GatsbyImageSharpFluid
        }
      }
    }
  }
}
```

Note that I don't have to do anything else, just add the `imageSharp` part to the query and development environment just silently stops working. At this point full builds still work fine. However as soon as I try to reference the imageSharp values in my app, and run a full build again, builds also break with `WebpackError: TypeError: Cannot read property 'fluid' of null`

```jsx
{section.imageSharp.childImageSharp.fluid && (
  <Img fluid={section.imageSharp.childImageSharp.fluid} />
)}
```

However, if I remove that and just render out `section.imageSharp` into pre tags, I get exactly what I'm expecting - an object containing the correct `childImageSharp.fluid` values.

If I run a query in GraphiQL which checks to see if Prismic is returning the values, I also get results:

<img width="1060" alt="Screenshot 2019-09-06 at 12 07 35" src="https://user-images.githubusercontent.com/189707/64423662-f415f280-d09e-11e9-9762-885e634d3e27.png">

I'll try and reduce this down to a repo to reproduce, but is there anything completely obvious that I've got wrong here? Any help greatly appreciated!

Have cleared and reinstalled node_modules and cache folders.

Dependency versions:
```
"gatsby": "^2.13.73",
"gatsby-image": "^2.2.10",
"gatsby-plugin-sharp": "^2.2.20",
"gatsby-source-filesystem": "^2.1.11",
"gatsby-source-prismic-graphql": "^3.4.0-beta.1",
"gatsby-transformer-sharp": "^2.2.13",
```

gatsby-config:
```js
{
  resolve: `gatsby-source-prismic-graphql`,
  options: {
    repositoryName: 'xxxxx',
    path: '/preview',
    previews: true,
    omitPrismicScript: true,
    pages: [
      {
        type: 'Landingpage',
        match: '/:uid',
        path: '/',
        component: require.resolve('./src/templates/landing.js'),
      },
    ],
  },
},
{
  resolve: `gatsby-source-filesystem`,
  options: {
    name: `images`,
    path: `${__dirname}/src/images`,
  },
},
`gatsby-transformer-sharp`,
`gatsby-plugin-sharp`,
```

----

Edit:

So this seems to be an issue with using the query on a page, rather than the landing template. Pages at `localhost:8000/:uid` build successfully and can be accessed on develop, however my homepage, in `/pages/index.js`, which just grabs a specific landing page and renders it as the root page (allowing us to easily swap out which landing page is used as the homepage just by changing the value) is the one that doesn't work. 

```js
query HomepagePageQuery {
  prismic {
    landingpage(lang: "en-gb", uid: "homepage") {
      body {
        ...
      }
    }
  }
}
```

**Comment by jodiedoubleday** [comment]
Did you ever find a fix for this using pages rather than templates? @benhodgson87 ?

**Comment by benhodgson87** [comment]
No, solution was to just not use sharp for images unfortunately. Not sure if this is a general bug with trying to use sharp on pages, or just due to the way I was using pages. If it is a bug with pages this should probably be reopened.

**Comment by amihalopoulos** [comment]
I am experience this same issue as well
